packages/pages/src/common/src/template/internal/getTemplateFilepaths.ts

```diff
@@ -98,22 +91,11 @@ export const getGlobalClientServerRenderTemplates = (
  * @returns the path to the appropriate file along with a boolean denoting if the path returned is a custom render template
  */
 const findGlobalRenderFile = (
-  templatesRootPath: Path,
-  scopedTemplatePath: Path | undefined,
+  templatePaths: Path[],
   globalFilename: string
 ): [string, boolean] => {
-  if (scopedTemplatePath) {
-    const pathToGlobalFile = path.join(
-      scopedTemplatePath.getAbsolutePath(),
-      globalFilename
-    );
-    if (fs.existsSync(pathToGlobalFile)) {
-      return [pathToGlobalFile, true];
-    }
-  }
-
   const pathToGlobalFile = path.join(
-    templatesRootPath.getAbsolutePath(),
+    templatePaths[0].getAbsolutePath(),
```

Comment:
hm it's not clear to me why we just choose the first path here.

Reply:
TAL at `getTemplatePaths` in structure.ts. If there's a scope defined it returns [the scoped templates, non-scoped templates]. If no scope is defined it's just [non-scoped templates]. The 0 index always gives us the appropriate set of templates, whether scoped or not.